src/pointfree/index.js

```diff
@@ -39,6 +39,7 @@ module.exports = {
   snd: require('../Pair/snd'),
   swap: require('./swap'),
   tail: require('./tail'),
+  toPromise: require('./toPromise'),
```

Comment:
This will need to point to `'../Async/toPromise.js'`, when it is moved over to the `Async` folder.